shared/video-sdk/get-started/get-started-sdk/project-implementation/unreal.mdx

```diff
@@ -133,25 +154,29 @@ To implement this interface, do the following:
 
         A node appears in **Level Blueprint**.
 
-    1. Use the same procedure to add the **Add to Viewport** node to **Level Blueprint**.
+    1.  Add the **Add to Viewport** node to **Level Blueprint**:
 
-    1. Connect **Event BeginPlay** to the left connectors of the **Construct None** node.
+            1. Drag the right pin of the **Construct NONE** node to the level editor, then right-click the dragged pin end and search for **Add to Viewport**. From the search results, choose **Add to Viewport**.
```

Comment:
@Pankajg123 In the step 6/ii we add a node but don't say what it is called: 

> Right-click in the Event Graph window and input create widget in the Search field. From the search results, choose Create Widget.

But then in step 6/iii we suddenly have a Construct NONE node. 

Is this the node we added in the previous step? If so, let's specify its name in step 6/ii. Also while the procedure says 
Construct NONE, the screenshot does not say Construct NONE, it says Create New Blueprint Widget instead (if I understand this correctly). This can be confusing for the reader.

Reply:
Thanks Incorporated your suggestion. Put about `Construct NONE` in 6/ii.
As we follow the step 6/iv  (Set the Class field of the Construct NONE node to NewBlueprint.)  then `Construct NONE` title change as per the value of setting Clsss field. So the  title changed from **Construct NONE** to  **Create New Blueprint Widget** as class is set to **New Blueprint**.

Please check  (2 - Adding the UI Blueprint Widget to your Level) in below link:
https://docs.unrealengine.com/4.27/en-US/InteractiveExperiences/UMG/HowTo/CreatingWidgets/